md_rahaman/python/django/survey_form/apps/survey_form_app/templates/survey_form_app/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
```

Comment:
The tabbing and spacing in this file is perfect. Keep up this type of format across all your code.